packages/gestalt/src/SideNavigation/ItemsEllipsis.js

```diff
@@ -0,0 +1,59 @@
+// @flow strict
+import { type Node as ReactNode } from 'react';
+import Box from '../Box';
+import { useSideNavigation } from '../contexts/SideNavigationProvider';
+import Icon from '../Icon';
+import { type Props as TopItemProps } from '../SideNavigationTopItem';
+import TapArea from '../TapArea';
+
+export type Props = {
+  ...Pick<TopItemProps, 'active' | 'notificationAccessibilityLabel'>,
+  accessibilityLabel?: string,
+};
+
+export default function ItemsEllipsis({
+  active,
+  accessibilityLabel = 'Collapsed navigation items. Expand for more options',
+  notificationAccessibilityLabel,
+}: Props): ReactNode {
+  const { setOverlayPreview } = useSideNavigation();
+
+  const itemColor = active === 'page' ? 'selected' : undefined;
+  const textColor = active === 'page' ? 'inverse' : 'default';
+
+  return (
+    <TapArea
+      accessibilityLabel={accessibilityLabel}
+      rounding={2}
+      onTap={() => setOverlayPreview(true)}
+    >
+      <Box
+        width={44}
+        height={44}
+        rounding={2}
+        display="flex"
+        justifyContent="center"
+        alignItems="center"
+        color={itemColor}
+        position="relative"
+      >
+        {notificationAccessibilityLabel ? (
+          <Box
+            aria-label={notificationAccessibilityLabel}
+            height={8}
+            width={8}
+            rounding="circle"
+            color="primary"
+            role="status"
+            position="absolute"
+            dangerouslySetInlineStyle={{ __style: { top: 4, right: 4 } }}
+          />
+        ) : null}
+
+        <Box>
```

Comment:
Do we need this empty Box?

Reply:
Yes. I added it so that icon doesn't get squished or smaller when ellipsis has border and border shrinks the content width.